astronomer/providers/amazon/aws/example_dags/example_emr.py

```diff
@@ -0,0 +1,82 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.providers.amazon.aws.operators.emr import EmrContainerOperator
+
+from astronomer.providers.amazon.aws.sensors.emr import EmrContainerSensorAsync
+
+# [START howto_operator_emr_eks_env_variables]
+VIRTUAL_CLUSTER_ID = os.getenv("VIRTUAL_CLUSTER_ID", "astro-emr-cluster")
+AWS_CONN_ID = os.environ.get("ASTRO_AWS_CONN_ID", "aws_default")
```

Comment:
why not os.getenv for AWS_CON_ID as well?

Reply:
Both works, now fixed to `os.getenv`